test_scripts/API/VehicleData/StabilityControlsStatus/008_GetVehicleData_StabilityControlsStatusNotAllowedByPolicy.lua

```diff
@@ -0,0 +1,55 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: Check processing of disallowed by policy stabilityControlsStatus data
+-- via GetVehicleData RPC
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- 3) PTU is successfully performed, stabilityControlsStatus NOT allowed by Policies
+--
+-- Steps:
+-- 1) App sends GetVehicleData in (NONE, FULL, LIMITED, BACKGRAUND) level (with stabilityControlsStatus = true) request to SDL
+--  - SDL not sends VehicleInfo.GetVehicleData (with stabilityControlsStatus = true) request to HMI
+--  - SDL sends VehicleInfo.GetVehicleData response ( resultCode = DISALLOWED, success = false )
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/VehicleData/commonVehicleData')
+local runner = require('user_modules/script_runner')
+
+--[[ Local Variables ]]
+local resultCode = "DISALLOWED"
+
+--[[ Local Functions ]]
+function ptUpdateMin(pTbl)
+  pTbl.policy_table.app_policies[common.getConfigAppParams().fullAppID].groups = { "Base-4", "Emergency-1" }
+  local grp = pTbl.policy_table.functional_groupings["Emergency-1"]
+  grp.rpcs.GetVehicleData.hmi_levels = {
+    "FULL",
+    "LIMITED",
+    "BACKGROUND",
+    "NONE" 
+  }
+  grp.rpcs.GetVehicleData.parameters = common.EMPTY_ARRAY
```

Comment:
@ydementieiev Please disallow only `stabilityControlsStatus` parameter, not all of them.